src/main/java/no/unit/nva/database/DatabaseServiceImpl.java

```diff
@@ -28,12 +29,18 @@
 import nva.commons.utils.JacocoGenerated;
 import nva.commons.utils.SingletonCollector;
 import nva.commons.utils.attempt.Failure;
+import nva.commons.utils.attempt.Try;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class DatabaseServiceImpl extends DatabaseServiceWithTableNameOverride {
 
-    public static final String RANGE_KEY_NAME = "PK1B";
+    public static final String SECONDARY_INDEX_1_HASH_KEY = "SecondaryIndex1HashKey";
```

Comment:
index names and key names cannot be loaded in runtime. They should be available in compilation time. Collect them in one class for better overview